.circleci/config.yml

```diff
@@ -117,6 +117,14 @@ jobs:
       - run: yarn run example:build:prod
       - run: yarn run example:cypress:ci
 
+  docs-tests:
```

Comment:
I believe the initial setup should match lines 129-135 so the cache is restored correctly